client_interface.py

```diff
@@ -44,6 +44,7 @@ def send_to_server(self, message):
         """
         Client connects to server and receives messages.
         """
+        global robot_name
```

Comment:
Proč je zde `global`?

Reply:
to budu ráda pokud poradíte jak to jinak napsat, protože když server na začátku pošle jen robot_name, nastaví se to pro tu zprávu `elif "robot_name" in message:` jenže to potom házalo chybu v ` self.set_game_state(message)` že robot_name není definováno.

Reply:
Jedna z možností, aby funkce pracovala s nějakými vstupními daty, je použit tato data jako argument funkce. Další, která mě napadá, je použít atribut třídy. Protože ale součástí interface je samotný robot, duplikovaly bychom tím data, šla bych tedy cestou 1.

Reply:
opraveno, snad jsem to pochopila jak :)